tests/Dialect/Jaxite/IR/ops.mlir

```diff
@@ -34,4 +34,22 @@ module {
     %out = jaxite.lut3 %e1, %e2, %e3, %truth_table, %bsks, %params : (!ciphertext, !ciphertext, !ciphertext, i8, !bsks, !params) -> !ciphertext
     return
   }
+
+  // CHECK-LABEL: func @test_packed_lut3
+  func.func @test_packed_lut3(%bsks : !bsks, %params : !params) {
+    %0 = arith.constant 1 : i1
+    %1 = arith.constant 1 : i1
+    %2 = arith.constant 1 : i1
+    %truth_table = arith.constant 8 : i8
+
+    %e1 = jaxite.constant %0, %params : (i1, !params) -> !ciphertext
+    %e2 = jaxite.constant %1, %params : (i1, !params) -> !ciphertext
+    %e3 = jaxite.constant %2, %params : (i1, !params) -> !ciphertext
+
+    %l1 = jaxite.lut3_args %e1, %e2, %e3, %truth_table :(!ciphertext, !ciphertext, !ciphertext, i8) -> !jaxite.pmap_lut3_tuple
+    %lut3_args_tensor = tensor.from_elements %l1 : tensor<1x!jaxite.pmap_lut3_tuple>
```

Comment:
this is just a suggestion since something working is better than nothing.

why not define `lut3_args` to take the entire list of lut arguments and return a tensor of lut3 tuples? that way you avoid having to specially handle `tensor.from_elements` in the emitter and you won't have this second layer of indirection when you are trying to grab the pmap_lut3's operands

In the tablegen you could have the input be `Variadic<AnyType>`, create a verifier that ensures that every argument is a lwe ciphertext but every fourth argument which is a truth table int and then create a printer that prints the argument as: 

```
    %lut3_args_tensor = jaxite.lut3_args (%e1, %e2, %e3, %truth_table), (%e1, %e2, %e3, %truth_table) : tensor<2x!jaxite.pmap_lut3_tuple>
```

that lines up pretty close to how the pmap lut3 inputs are created in python

Reply:
I actually really like the lut3_args_tensor idea. Its much cleaner. Im thinking of doing that as a follow-up if its ok. 
Created an issue for it for tracking: https://github.com/google/heir/issues/1075

Is there a tag for clean-up type issues?

Reply:
you can always make one :)